apps/web/vibes/soul/primitives/cursor-pagination/index.tsx

```diff
@@ -26,9 +25,9 @@ interface Props {
 
 export function CursorPagination(props: Props) {
   return (
-    <Suspense fallback={<CursorPaginationSkeleton />}>
-      <CursorPaginationResolved {...props} />
-    </Suspense>
+    <Stream fallback={<CursorPaginationSkeleton />} value={props}>
+      {CursorPaginationResolved}
+    </Stream>
```

Comment:
This isn't the proper use of `Stream` as it's calling `CursorPaginationResolved` instead of rendering it as a component, which would break the rules of hooks, have issues with RSC/client component boundaries, etc. Also, `value` should be a streamable value, not all props.

Instead, we should use leave this unchanged as the `useStreamable` calls in `CursorPaginationResolved` will suspend, triggering the Suspense boundary and fallback.